measure.html

```diff
@@ -38,10 +38,18 @@ <h1>Experiment Details</h1>
         <label>Brave Email ID (username@brave.com):</label><br>
         <input type="text" placeholder="username@brave.com" name="email"><br />
         <label>Are you connecting to us through:</label><br>
-        <input type="radio" id="exp_type" name="exp_type" value="direct">
+        <input type="radio" id="exp_type" name="exp_type" value="direct" onclick="document.getElementById('locinfo').style.display='none'">
         <label for="direct">Direct Connection</label><br>
-        <input type="radio" id="exp_type" name="exp_type" value="vpn">
+        <input type="radio" id="exp_type" name="exp_type" value="vpn" onclick="document.getElementById('locinfo').style.display='block'">
         <label for="vpn">VPN</label><br>
+        <div id="locinfo" style="display:none;">
+          <label>Location of VPN server connected (enter city, state):</label><br>
+          <input type="text" name="location_vpn"><br />
+          <label>Location of user (enter city, state):</label><br>
```

Comment:
Some rephrasing to make it clear that we're asking about the user's location.
```suggestion
          <label>Your location:</label><br>
```

Reply:
Done in 0da43b5